torchgeo/datamodules/treesatai.py

```diff
@@ -72,7 +72,7 @@ class TreeSatAIDataModule(NonGeoDataModule):
     def __init__(
         self,
         batch_size: int = 64,
-        patch_size: int | tuple[int, int] = 304,
+        patch_size: int | tuple[int, int] = (304, 304),
```

Comment:
This is unnecessary

Reply:
It's not because by default if you only pass 304 as an int it would do shortest side resizing. Are you saying that the dataset itself is okay to do this?

Reply:
> This is unnecessary

It's not because by default if you only pass 304 as an int it would do shortest side resizing. Can you explain why it's not necessary? Here and below.